### ISSUE TITLE
Добавь возможность убирать сайты, если они "Переобулись"

### ISSUE THREAD

**Issue by CoomerRussian** [stravnik/toxic-repos#96]
P.S Мегахорош Братан

**Comment by stravnik** [comment]
@CoomerRussian этот вопрос уже поднимался. Мы не ведем список "прям здесь и сейчас", попасть в список легко, а выйти из него невозможно.
Пусть сообщество видит и помнит этих гон*ов.

**Comment by OLEG4120** [comment]
Да, в таком случаи делаете PR, что кто-то переобулся, мы люди все равно должны знать этих засранцев

**Comment by CoomerRussian** [comment]
> @CoomerRussian этот вопрос уже поднимался. Мы не ведем список "прям здесь и сейчас", попасть в список легко, а выйти из него невозможно. Пусть сообщество видит и помнит этих гон*ов.

Понятно.

**Comment by lightmanLP** [comment]
Не понимаю что именно непростительного сделали продукты, которые содержат исключительно слоганы/флаг.

**Comment by stravnik** [comment]
@lightmanLP сам факт наличия слогана/флага себя дискредитировал неоднократно. Поэтому наличие косвенных признаков теперь приводит к попаданию в список.

**Comment by vitich** [comment]
Сам факт наличия в гитхабе репозитория который поддерживают представители страны-террориста (временное недоразумение) - вот это дискредитация опенсорс. Ну ничего, мы это исправим и вы сходите вслед за своим крейсером москва. А по поводу флагов и support-ukraine - вас теперь лет 200 минимум будут на бутылку присаживать всюду только за то что вы - россияне.

**Comment by vitich** [comment]
Ну ничего, мы тебя пробьём и твою контору найдём и пойдёшь репу жрать в стойло.

**Comment by vitich** [comment]
Про "великую державу" сильно )))

**Comment by vitich** [comment]
> @vitich уже пробивали и звонили и угрожали и т.д. и т.п. Диалог окончен, ты неинтересен.

Видишь ли... мы только начали.

**Comment by stravnik** [comment]
> > @vitich уже пробивали и звонили и угрожали и т.д. и т.п. Диалог окончен, ты неинтересен.
> 
> Видишь ли... мы только начали.

Уже год "только начинаете" =)

**Comment by vitich** [comment]
И это, Алексей, или вас на*бывают или вы демпингуете... Нехорошо. Сколько же вы в своём АО "ГНИВЦ" зарабатываете, подумать страшно, если с такой гордостью про оффер на $2.5к...

**Comment by vitich** [comment]
> @vitich давно тут не было представителей салорейха. я тебя расстрою, но практика показывает, что представителей великой державы очень любят работодатели. Вот только сегодня получил оффер от зарубежной компании на *2,5 ЗП.

И это, Алексей, или вас на*бывают или вы демпингуете... Нехорошо. Сколько же вы в своём АО "ГНИВЦ" зарабатываете, подумать страшно, если с такой гордостью про оффер на $2.5к...

**Comment by stravnik** [comment]
@vitich А что-то новое расскажите? Ваша инфа устарела немного =)
в ГНИВЦе я уже не работаю, но т.к. я это не афишировал, то инфы у вас нет :D

**Comment by empty-git** [comment]
Друзі та вороги, дякую що створили подібний репозиторій.
Скарга вже надіслана на модерацію до проєкту Github.
Сподіваюсь що війна скоро закінчиться і росіяни почнуть платити репарації, а ви будете сидіти як люди низького інтелекту та жалітися на те як вам тяжко.

**Comment by lightmanLP** [comment]
Репозиторий превращается в хохлосрач с геометрической прогрессией.

> @lightmanLP сам факт наличия слогана/флага себя дискредитировал неоднократно. Поэтому наличие косвенных признаков теперь приводит к попаданию в список.

Репозиторий же изначально предполагался как список потенциально **опасных** репозиториев. При этом малварей в нем всего 8, а в `hostile_actions` входят ушедшие из рф сайты и смс бомберы.
